inc/ticket.class.php

```diff
@@ -4036,7 +4035,7 @@ function showFormHelpdesk($ID, $ticket_template = false) {
          echo "<td>".sprintf(__('%1$s%2$s'), __('Title'), $tt->getMandatoryMark('name'))."<td>";
          if (!$tt->isHiddenField('name')) {
             echo "<input type='text' maxlength='250' size='80' name='name'
```

Comment:
Here, to avoid code replication, maybe use 

```php
echo Html::input('name', [
   'value'     => $options['name'],
   'maxlenght' => 250,
   'size'      => 80
];
```

Reply:
I will change

Reply:
@orthagh, I noticed that there are more inputs like this, can I create a separate pull?

Reply:
for master branch, yes, for 9.2/bf, just here

Reply:
Because of `$tt->isMandatoryField('name')`, I can change Html class?

```php
// in ticket.class.php
echo Html::input('name', [
    'value'     => $options['name'],
    'maxlength' => 250,
    'size'      => 80,
    'required'  => $tt->isMandatoryField('name'),
]);


//in html.class.php
   static function formatAttribute($key, $value) {
      // begin new code
      if (is_bool($value)) {
         if ($value) {
            return sprintf('%1$s="%2$s"', $key, $key); //Ex. required="required"
         }
         return '';
      }
      // end new code
      if (is_array($value)) {
         $value = implode(' ', $value);
      }

      return sprintf('%1$s="%2$s"', $key, Html::cleanInputText($value));
   }
```

Reply:
for the modification, only if it's mandatory.
I suggest to prepare the ``$options`` array before and append required in an ``if`` statement

Reply:
Modifying keeps code cleaner. The Yii2 framework looks like this.

For master branch, I will make